File: Data Structure/Lab 7/NodeStructure.c
//Write a program to implement a node structure for singly linked list. Read the 
//data in a node, print the node. 

#include<stdio.h>
#include<stdlib.h>

struct Node
{
    int info;
    struct Node* link;
};

void main(){

    struct Node* node = (struct Node*)malloc(sizeof  (struct Node*));

    if(node == NULL){
        printf("Memory Allocation Failed!");
        return;
    }

    printf("Enter data of node:");
    scanf("%d",&(node -> info));

    node -> link = NULL;

    printf("Node data : %d",node -> info);

    free(node);

}

File: Data Structure(Java)/Lab 5/NodeStructure.java
// Write a program to implement a node structure for singly linked list. Read the 
//data in a node, print the node. 

class Node {

    int info;
    Node link;

    Node(int info){
        this.info = info;
        this.link = null;
    }
}

public class NodeStructure {
    public static void main(String[] args) {
        
        Node node1 = new Node(1);

        System.out.println(node1.info);
        System.out.println(node1.link);
    }
}
